content/2.terms/1.community guidelines.md

```diff
@@ -0,0 +1,21 @@
+# Community Guidelines
+
+## Overview
+Our goal is to foster an inclusive, respectful, and engaging space where everyone can express themselves openly and connect with others around the globe. These community guidelines will outline which type of content is not allowed.
+
+
+
+### Spam
+This refers to unsolicited, repetitive messages sent en masse and without meaningful context or content.

```

Comment:
How do you feel about including "advertising", in the sense of actual product shilling/advertising from companies? Is it even something people agree with?

Reply:
I think advertising is okay as long as its meaningful. 

Like if I make custom auto parts for a niche car the community around that niche car would find value in my post advertising my services/business.